src/DonationForms/Blocks/DonationFormBlock/resources/editor/Edit.tsx

```diff
@@ -93,7 +105,16 @@ export default function Edit({attributes, isSelected, setAttributes, className,
                     )}
                 </>
             ) : (
-                <DonationFormSelector formOptions={formOptions} isResolving={isResolving} handleSelect={handleSelect} />
+                <EntitySelector
```

Comment:
We do also have the  [CampaignSelector](https://github.com/impress-org/givewp/blob/8664aa64cfe027fc6c418fd2440e2e777667ffac/src/Campaigns/Blocks/shared/components/CampaignSelector/index.tsx#L7) component that is context aware of the campaign page.